code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -146,7 +133,6 @@
 	hats["hat, bowler"] = /obj/item/clothing/head/bowler
 	hats["hat, fez"] = /obj/item/clothing/head/fez
 	hats["hat, tophat"] = /obj/item/clothing/head/that
-	hats["hat, feather trilby"] = /obj/item/clothing/head/feathertrilby
```

Comment:
A strange removal that I don't support. Most of our hat sprites aren't up to par I know, but this one offers something unique from the others, and is far from the worst.

Reply:
Agree with fernerr